README.md

```diff
@@ -1,7 +1,8 @@
 # Vinyl Music Player
 [![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](https://github.com/VinylMusicPlayer/VinylMusicPlayer/blob/master/LICENSE.txt) 
-[![Android CI](https://github.com/VinylMusicPlayer/VinylMusicPlayer/actions/workflows/ci.yml/badge.svg)](https://github.com/VinylMusicPlayer/VinylMusicPlayer/actions/workflows/ci.yml)
+[![GithubActions CI](https://github.com/VinylMusicPlayer/VinylMusicPlayer/actions/workflows/ci.yml/badge.svg)](https://github.com/VinylMusicPlayer/VinylMusicPlayer/actions/workflows/ci.yml)
 
+[![Download](https://img.shields.io/badge/Download-@pre--release-green)](https://github.com/vinyl2-team/vinyl2/releases/tag/pre-release)
```

Comment:
I think the link needs to be updated to reflect the new repo URL

Reply:
Done, thanks